src/tpt_reports/tpt_reports.py

```diff
@@ -67,12 +66,13 @@ def parse_json(data):
     payloads_meta = {}
     try:
         if data:
-            for payload in data["payloads"]:
+            assessment_id = data["id"]
```

Comment:
Is it assumed that this updated code will never be run on the older JSON files?  Generally, in a situation like this, I'd use something like `assessment_id = data.get("id", "N/A")` to handle the possibility of a missing `id` field (e.g. if this were run with the older-style JSON).

Reply:
Operators do not expect that the format would revert, but the suggestion was applied in f04114a31e9a9b8f3a2176880248a7ee2e15fa5d. 

If for whatever reason the ID is not provided, we would at least have a fallback value to apply.

Reply:
Looks good! 👍